$NetBSD: patch-ab,v 1.1 2006/05/18 18:16:06 rillig Exp $

SunPro cannot resolve the overloading conflict between sqrt(double) and
sqrt(long double) when provided with an integer.

--- SpielfeldAufbau.cpp.orig	2006-05-18 20:07:27.187451459 +0200
+++ SpielfeldAufbau.cpp	2006-05-18 20:06:35.247252233 +0200
@@ -26,7 +26,7 @@
 
     GLfloat r=5.715;
     GLfloat a=r+3*Epsilon;
-    GLfloat asw3=a*sqrt(3)/6;
+    GLfloat asw3=a*sqrt(3.0)/6;
     
     srand(Zeit());
     
